openreview/tools.py

```diff
@@ -703,3 +698,39 @@ def post_submission_groups(client, conference_id, submission_invite, chairs):
             readers=[conference_id, chairs],
             signatories=[]))
 
+def get_submission_invitations(client, status='all'):
+    '''
+    Returns a list of invitations ids according to the status.
+    
+    :arg client: Object of :class:`~openreview.Client` class
+    :arg status: Status filter for invitations. Default value is "all". This is a string with following possible value: all, open and closed.
+    '''
+    
+    try:
+        # Get a list of all group with invitations satisfying the regex
+        groups = client.get_invitations(regex='.*/-/.*submission')
+        
+        # For each group in the list, append the group ids to a list
+        invitation_ids = []
+        
+        for g in groups :
+            
+            #Calculate the epoch for current timestamp
+            now = int(time.time()*1000)
+
+            # Assume the invitation is closed unless proven open
+            due = 0
+
+            if g.to_json()['duedate'] != None and now < g.to_json()['duedate']:
```

Comment:
no need to convert to json to get the duedate

Reply:
Corrected